Sources/TensorFlow/Core/PythonConversion.swift

```diff
@@ -12,13 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if canImport(Python)
+#if canImport(Python) || canImport(PythonKit)
+
+#if canImport(PythonKit)
+import PythonKit
+#else
 import Python
+#endif
+
 
 /// The `numpy` Python module.
 /// Note: Global variables are lazy, so the following declaration won't produce
 // a Python import error until it is first used.
+#if canImport(PythonKit)
+private let np = PythonKit.import("numpy")
```

Comment:
The `Python` in `Python.import` here refers to [a global variable](https://github.com/pvieito/PythonKit/blob/c699071262161e2f2d9fa081d9cc8ef31ca4ec5f/PythonKit/Python.swift#L640), not a module name.

```suggestion
private let np = Python.import("numpy")
```

Same below: `PythonKit.isinstance` -> `Python.isinstance`.

I believe the only necessary change in this patch is the top-level ifdef'd import declaration!